src/js/components/leftPanel/LeftPanel.tsx

```diff
@@ -81,20 +111,60 @@ interface TabsProps {
 }
 
 const Tabs = (props: TabsProps): React.ReactElement => {
+  const [documentFlashing, setDocumentFlashing] = useState(false);
+  const [analysisFlashing, setAnalysisFlashing] = useState(false);
   //Active Tab in the store
   const savedActiveTab = useSelector(
     (store: RootState) => store.appState.leftPanel.activeTab
   );
 
-  const tabsGroupRow = props.tabsToRender.map(tab => (
-    <Tab
-      key={tab.label}
-      label={tab.label}
-      tooltipText={tab.tooltipText}
-      icon={tab.icon}
-      activeTab={savedActiveTab}
-    />
-  ));
+  const activeFeatures = useSelector(
+    (store: RootState) => store.mapviewState.activeFeatures
+  );
+
+  const activeFeatureIndex = useSelector(
+    (store: RootState) => store.mapviewState.activeFeatureIndex
+  );
+
+  const [featureCollectionIndex, featureIndex] = activeFeatureIndex;
+
+  useEffect(() => {
+    const specificFeature =
+      activeFeatures[featureCollectionIndex]?.features[featureIndex];
```

Comment:
no sure why we need to look for that active feature here? Can we simply check if `activeFeatures` length is larger than 0?

Reply:
I know we chatted about this yesterday, but I wanted to document this for future reference - this logic helps track when the user selects 'next' or 'previous' in the data tab. This way the doc/analysis tab can render the 'pulsing' effect with each selection